pkg/microservice/systemconfig/core/codehost/handler/codehost.go

```diff
@@ -77,7 +77,7 @@ func GetCodeHost(c *gin.Context) {
 		ctx.Err = err
 		return
 	}
-	ctx.Resp, ctx.Err = service.GetCodeHost(id, ctx.Logger)
+	ctx.Resp, ctx.Err = service.GetCodeHost(id, c.Query("ignoreDelete") == "true", ctx.Logger)
```

Comment:
Is it better to use `stroconv.ParseBool()` to support values of `ignoreDelete`?
![image](https://user-images.githubusercontent.com/829039/169767847-ac4c026e-4f3b-4e04-b365-3c89b9b1a623.png)